tests/unit/replaceContent.test.ts

```diff
@@ -54,13 +59,14 @@ describe('replaceContent', () => {
 			<div id="container-1" data-from="current"></div>
 		`);
 		const warn = vi.spyOn(console, 'warn');
-		const page = mockPage(/*html*/ `
+		const html = `
```

Comment:
The `/* html */` comment tells this vscode extension to correctly syntax-highlight the html template literal:

https://marketplace.visualstudio.com/items?itemName=Tobermory.es6-string-html

Reply:
Good to know! I've added them back.